.github/workflows/kubectl-aks.yml

```diff
@@ -268,6 +268,39 @@ jobs:
           export AZURE_NODE_NAME=$(echo '${{ needs.create-aks-cluster.outputs.nodes }}' | jq -r ".[${{ strategy.job-index }}]")
           make integration-test -o kubectl-aks
 
+  documentation-test:
+    name: Run documentation tests
+    runs-on: ubuntu-latest
+    needs: [ build, unit-tests ]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '>=1.20.0' # ie requires go 1.20+
+          check-latest: true
+      - name: Get kubectl-aks from artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: kubectl-aks-linux-amd64-tar-gz
+      - name: Prepare kubectl-aks binary
+        shell: bash
+        run: |
+          tar zxvf kubectl-aks-linux-amd64.tar.gz
+          chmod +x kubectl-aks
```

Comment:
Mmmmm ... Why it doesn't have this permissions already? I don't remember doing this before.